build/less/container.less

```diff
@@ -0,0 +1,39 @@
+// Container widths
```

Comment:
I am really against .container. 

Sure, it is faster and easier to have, but it leads to less use of semantic class names. See http://w3c.github.io/html/dom.html#global-attributes:

> There are no additional restrictions on the tokens authors can use in the class attribute, but authors are encouraged to use values that describe the nature of the content, rather than values that describe the desired presentation of the content.

Further reading: https://tympanus.net/codrops/2013/01/22/defending-presentational-class-names/

In summary: While it is sometimes useful to create reusable class-names that describe a certain, heavily reused style, the `container` class is certainly not one of them.